docs/installation.mdx

```diff
@@ -123,4 +123,33 @@ Version:        1.0.0
 Go version:     go1.20.2
 Git commit:     b58e7b910ca556973d111e9bd734a71baef03db2
 Git tree state: clean
+```
```

Comment:
Sajay made a suggest about creating another page for this which I think is good because we don't need people to go through this unless they want to.

Reply:
@TerryHowe I think we could add it to the developer's guide. The developer's guide already has the `release checklist`.

Reply:
Maybe Sajay has another idea, but maybe just a how-to and link it here?